source/game/system/map/MapdataCheckPath.hh

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "game/system/map/MapdataAccessorBase.hh"
+
+#include <egg/math/Vector.hh>
+
+namespace System {
+class MapdataCheckPath {
+public:
+    struct SData {
+        u8 start;
+        u8 size;
+
+        u8 prev[6];
+        u8 next[6];
+        u16 _padding;
```

Comment:
```suggestion
```

Reply:
This breaks the static assert. If the static assert is commented out, then it causes a segfault down the line because it doesn't read the values correctly.